Python/pystate.c

```diff
@@ -280,11 +280,19 @@ _PyInterpreterState_DeleteExceptMain()
 
     HEAD_LOCK();
     PyInterpreterState *interp = _PyRuntime.interpreters.head;
+    PyInterpreterState *tmp_interp = NULL;
     _PyRuntime.interpreters.head = NULL;
-    for (; interp != NULL; interp = interp->next) {
+    fprintf(stdout, "1. interp: %p\n", interp);
+    fprintf(stdout, "2. interp->next: %p\n", interp->next);
+    fprintf(stdout, "2.1. _PyRuntime.interpreters.main: %p\n", _PyRuntime.interpreters.main);
```

Comment:
Remove debug printf.

Reply:
(Those have been pushed recently to diagnose the "take infinite time to build when this code is added" / "make CI timeout" issue)

Reply:
@vstinner yep, just for the debug and because I wanted to show you